README.md

```diff
@@ -6,9 +6,9 @@ Keeping the trash in your ./vendor dir to a minimum.
 
 Make sure you're using go1.6 or later version.
 
- 0. Download and extract `trash` to your PATH
+ 1. Download and extract `trash` to your PATH: `go get github.com/rancher/trash`.
```

Comment:
`go get` totally works, but there are actual binary releases to download.

So, maybe also add a link to https://github.com/rancher/trash/releases ?